src/v/raft/tests/state_removal_test.cc

```diff
@@ -62,6 +62,7 @@ void stop_node(raft_node& node) {
         node._nop_stm->stop().get0();
     }
     node.raft_manager.stop().get0();
+    node.hbeats->stop().get0();
```

Comment:
can't tell which one is the real fix. is it stopping heartbeats or extending timeouts? 30 seconds is a long time.

Reply:
There are two different errors
1. in partition_movement_tests
2. snapshot deletion test

There are two commits.

Regarding the timeout, it is long, but we expect the test to exert different types of errors.